### DISCUSSION TITLE
Ingreso a FrontEnd

### DISCUSSION THREAD

**Discussion by rauoliv** [Launch-X-Latam/MisionFrontEnd#378] [General]
Buenas noches me quiero poner al dia con la parte de FrontEnd, alguien sabe cuanto tiempo toma en que aprueben el ingreso al teams de FrontEnd?

**Comment by Shian11** [comment]
> Buenas noches me quiero poner al dia con la parte de FrontEnd, alguien sabe cuanto tiempo toma en que aprueben el ingreso al teams de FrontEnd?  

También quiero ingresar al equipo de frontend de teams no pude ingresar antes

**Reply by rauoliv** [reply]
aun sigo esperando que aprueben  el ingreso al teams para el frontend.  @Shian11 ya te dieron acceso al teams de frontend?

**Reply by Shian11** [reply]
Todavía sigo esperando pero estoy mirando los vídeos de salen en los link de las clases por mientras
